Tasks/Kubernetes/src/utilities.ts

```diff
@@ -0,0 +1,90 @@
+"use strict";
+
+var https   = require('https');
+var fs      = require('fs');
+import * as path from "path";
+import * as tl from "vsts-task-lib/task";
+import * as os from "os";
+import * as util from "util";
+
+export async function download(url: string, downloadPath: string): Promise<void> {
+    var file = fs.createWriteStream(downloadPath);
```

Comment:
Would download require retries? Also, I see Packer task does a download (https://github.com/Microsoft/vsts-tasks/blob/master/Tasks/PackerBuild/src/utilities.ts). Would it be better to have this in Common and both the task use it? Adding @bishal-pdMSFT